parameters/tools/rviz_config.rviz

```diff
@@ -579,6 +579,7 @@ Visualization Manager:
       Entity Area label opacity: 0.400000006
       Entity Area opacity: 0.200000003
       Entity label opacity: 1
+      Exclude labels: hero
```

Comment:
Why is this (or is this also the case for the other robots)?

Reply:
I also created PRs for AMIGO and SERGIO. I was just really annoyed by the floating labels around the robots.

Reply:
See https://github.com/tue-robotics/ed_rviz_plugins/pull/4